src/Simulation/Simulators.Tests/ResourcesEstimatorTests.cs

```diff
@@ -101,5 +100,43 @@ public void ToTSVTest()
             Assert.Equal(2, cliffords.Length);
             Assert.Equal("2", cliffords[1]);
         }
+
+        /// <summary>
+        /// Verifies that for multiple separately traced operations, the final
+        /// statistics are cumulative.
+        /// </summary>
+        [Fact]
+        public void VerifyTracingMultipleOperationsTest()
+        {
+            ResourcesEstimator sim = new ResourcesEstimator();
+
+            Operation_1_of_2.Run(sim).Wait();
+            DataTable data1 = sim.Data;
+
+            Assert.Equal(1.0, data1.Rows.Find("CNOT")["Sum"]);
+            Assert.Equal(1.0, data1.Rows.Find("QubitClifford")["Sum"]);
+            Assert.Equal(1.0, data1.Rows.Find("T")["Sum"]);
+            Assert.Equal(0.0, data1.Rows.Find("R")["Sum"]);
+            Assert.Equal(0.0, data1.Rows.Find("Measure")["Sum"]);
+            Assert.Equal(2.0, data1.Rows.Find("Width")["Sum"]);
```

Comment:
This label changes after merging #267

Reply:
n/a anymore as abandoned column name changes in #267